lib/freedom_patches/fast_pluck.rb

```diff
@@ -51,7 +51,6 @@ def pluck(*column_names)
       relation = apply_join_dependency
       relation.pluck(*column_names)
     else
-      enforce_raw_sql_whitelist(column_names)
```

Comment:
Just to confirm - this is no longer needed?

Reply:
Correct! This method doesn’t exist anymore in Rails 6+ :slightly_smiling_face: